plugins/usbdmx/UsbDmxPlugin.cpp

```diff
@@ -91,13 +93,25 @@ string UsbDmxPlugin::Description() const {
 "----------------------------\n"
 "\n"
 "This plugin supports various USB DMX devices including the \n"
-"Anyma uDMX, Eurolite, Fadecandy, Sunlite USBDMX2 & Velleman K8062.\n"
+"Anyma uDMX, Eurolite, Fadecandy, Sunlite USBDMX2, Velleman K8062 "
+"& DMXControl Projects e.V. Nodle U1.\n"
 "\n"
 "--- Config file : ola-usbdmx.conf ---\n"
 "\n"
 "libusb_debug_level = {0,1,2,3,4}\n"
 "The debug level for libusb, see http://libusb.sourceforge.net/api-1.0/ .\n"
 "0 = No logging, 4 = Verbose debug.\n"
+"\n"
+"nodle_mode = {0,1,2,3,4,5,6,7}\n"
+"The mode for the Nodle U1 interface to operate in.\n"
+"0 - Standby\n"
+"1 - DMX In -> DMX Out\n"
```

Comment:
My gut feeling is this should be presented within OLA's preferences as a series of bools, rather than having to know which mode number you want.